support-frontend/app/controllers/Application.scala

```diff
@@ -158,3 +161,20 @@ class Application(
       Redirect("/" + path, request.queryString, MOVED_PERMANENTLY)
   }
 }
+
+object CSSElementForStage {
+
+  def apply(getFileContentsAsHtml: RefPath => Option[StyleContent], stage: Stage, cssPath: RefPath): Either[RefPath, StyleContent] = {
+    if (stage == Stages.DEV) {
+      Left(cssPath)
+    } else {
+      getFileContentsAsHtml(cssPath).fold[Either[RefPath, StyleContent]] {
+        SafeLogger.error(scrub"Inline CSS failed to load")
```

Comment:
Do we need this logging? If you think it's worth leaving in then maybe add the path to the message to help with debugging.

Reply:
that was already elsewhere, but actually you are right it would be better to send a metric to cloudwatch, and set up an alert to email us if that happens (as it means we broke something perf-wise)
I think I will add the path as you suggest, and we can revisit in a future PR regarding the alarm.

Reply:
ok I have pushed that change, I also found when testing locally it throws an assetnotfoundexception in some situations, which would cause a 500 error, so I have changed to a flatMap which should handle the option more gracefully.